components/server/src/liveness/liveness-controller.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { injectable, inject } from "inversify";
+import * as express from "express";
+import * as prom from "prom-client";
+import { Config } from "../config";
+import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
+
+@injectable()
+export class LivenessController {
+    @inject(Config) protected readonly config: Config;
+
+    get apiRouter(): express.Router {
+        const router = express.Router();
+        this.addLivenessHandler(router);
+        return router;
+    }
+
+    protected addLivenessHandler(router: express.Router) {
+        router.get("/", async (req, res, next) => {
+            const metrics = await prom.register.getMetricsAsJSON();
```

Comment:
Short-circuit the lookup if you do not have a `maximumEventLoopLag` value configured (< 0)

Reply:
done